Doucment `DOCS` build option in README

### Please tell us what you were doing! You can include code and files by drag and dropping them into the text area.

I was trying to build FluCoMa on macOS 14.7 against a current development version of SuperCollider.
Configuration threw an error, but finished:
```sh
cmake -D SC_PATH=/path/to/src/supercollider ..
-- The CXX compiler identification is AppleClang 16.0.0.16000026
...
-- Found Python: /opt/homebrew/Frameworks/Python.framework/Versions/3.13/bin/python3.13 (found version "3.13.5") found components: Interpreter
-- /opt/homebrew/Frameworks/Python.framework/Versions/3.13/bin/python3.13
-- Generating: DumpParams
/path/to/src/flucoma-sc/build/_deps/flucoma-docs-src/list_python_tests.py:11: SyntaxWarning: invalid escape sequence '\('
  parts = re.split("(.+)\((.+)\)",str(test))
Traceback (most recent call last):
  File "/path/to/src/flucoma-sc/build/_deps/flucoma-docs-src/list_python_tests.py", line 14, in <module>
    list_tests_from('flucoma/doc/test')
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/path/to/src/flucoma-sc/build/_deps/flucoma-docs-src/list_python_tests.py", line 10, in list_tests_from
    for test in suite:
                ^^^^^
TypeError: '_FailedTest' object is not iterable
-- Generating: FluidAmpFeature
...
-- Configuring done (4.6s)
-- Generating done (0.3s)
-- Build files have been written to: /path/to/src/flucoma-sc/build
```


### What was the expected result?

Build finishes.



### What was the actual result?

Build failed:
```sh
% make install
[  0%] Building CXX object _deps/fmt-build/CMakeFiles/fmt.dir/src/format.cc.o
...
[ 18%] Built target DumpParams
[ 18%] Making SC docs
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/path/to/src/flucoma-sc/build/_deps/flucoma-docs-src/flucoma/MakeRef.py", line 16, in <module>
    from flucoma.doc import render
  File "/path/to/src/flucoma-sc/build/_deps/flucoma-docs-src/flucoma/doc/render.py", line 14, in <module>
    from jinja2 import Environment, FileSystemLoader, select_autoescape
ModuleNotFoundError: No module named 'jinja2'
make[2]: *** [CMakeFiles/MAKE_SC_REF] Error 1
make[1]: *** [CMakeFiles/MAKE_SC_REF.dir/all] Error 2
make: *** [all] Error 2
```

I managed to build FluCoMa with `cmake -D SC_PATH=/path/to/src/supercollider -D DOCS=OFF ..` but I thought I'd report this.

### What operating system were you using?

Mac

### Operating system version

macOS 14.7

### FluCoMa Version

1.0.8 / main branch

--- a/README.md
+++ b/README.md
@@ -32,7 +32,25 @@ An alternative to setting up / running CMake directly on the command line is to 
 Also, with CMake you have a choice of which build system you use.
 
 - The default on macOS and Linux is `Unix Makefiles`. On macOS you can also use Xcode by passing `-GXcode` to CMake when you first run it.
-- The default on Windows is the latest version of Visual Studio installed. However, Visual Studio can open CMake files directly as projects, which has some upsides. When used this way, CMake variables have to be set via a JSON file that MSVC will use to configure CMake.
+- The default on Windows is the latest version of Visual Studio installed. However, Visual Studio can open CMake files directly as projects, which has some upsides. When used  this way, CMake variables have to be set via a JSON file that MSVC will use to configure CMake.
+## The `DOCS` option 
+The CMake option `DOCS` controls whether or not to generate the documentation for the FluCoMa objects. It is on by default. However, building the documentation is slightly involved and requires that you have a working Python 3 installation, among other things (see https://github.com/flucoma/flucoma-docs). 
+
+If you just want some binaries, it could be simpler to disable this and just take some documentation from a release download:
+
+```bash
+mkdir -p build && cd build
+cmake -DSC_PATH=</path/to/sc> -DDOCS=OFF ..
+make install
+```
+
+
+
+
+
+
+
+
 
 ## Using Manual Dependencies
 

--- a/README.md
+++ b/README.md
@@ -32,7 +32,7 @@ An alternative to setting up / running CMake directly on the command line is to 
 Also, with CMake you have a choice of which build system you use.
 
 - The default on macOS and Linux is `Unix Makefiles`. On macOS you can also use Xcode by passing `-GXcode` to CMake when you first run it.
-- The default on Windows is the latest version of Visual Studio installed. However, Visual Studio can open CMake files directly as projects, which has some upsides. When used  this way, CMake variables have to be set via a JSON file that MSVC will use to configure CMake.
+- The default on Windows is the latest version of Visual Studio installed. However, Visual Studio can open CMake files directly as projects, which has some upsides. When used this way, CMake variables have to be set via a JSON file that MSVC will use to configure CMake.
 ## The `DOCS` option 
 The CMake option `DOCS` controls whether or not to generate the documentation for the FluCoMa objects. It is on by default. However, building the documentation is slightly involved and requires that you have a working Python 3 installation, among other things (see https://github.com/flucoma/flucoma-docs). 
 

